charts/nodepool-labels-operator/templates/crd.yml

```diff
@@ -1,20 +1,33 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: nodepoollabelsets.labels.banzaicloud.io
-  labels:
-    app: {{ include "nodepool-labels-operator.name" . }}
-    chart: {{ include "nodepool-labels-operator.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
```

Comment:
**Question**: why are we removing these labels?

Reply:
fixed